x/wasm-storage/keeper/abci.go

```diff
@@ -109,3 +126,18 @@ func (k Keeper) ExecuteTally(ctx sdk.Context) error {
 
 	return nil
 }
+
+func tallyVMArg(reveals map[string]any, outliers []bool) ([]string, error) {
+	argBytes, err := rlp.EncodeToBytes(tallyArg{
```

Comment:
Should be JSON. From the Tally VM notion:

* REVEALS: An JSON encoded array of reveals coming from the contract. The shape of this JSON is out of scope of this document. Refer to the contract for an accurate format
* OUTLIARS: An JSON encoded array with the outliars that are calculated by the filter mechanism. Ex: [0, 0, 1, 1, 1, 1] Where the 1 means its considered correct and 0 means it was out of consensus.

Reply:
Here: [b12fb80](https://github.com/sedaprotocol/seda-chain/pull/280/commits/b12fb80e4341ab1eec6c1e5a0921759ffff97dfa)